dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -10,19 +11,20 @@ type Props = {
 };
 
 const Snap100 = ({
-	snap,
+	snaps,
 	containerPalette,
 	showAge,
 }: {
-	snap: TrailType;
+	snaps: TrailType[];
```

Comment:
👍 this isn't ideal _semantically_ but it is consistent

Reply:
You could possible have a tuple like `[TrailType]`?